dispatcher/MongoConnect.py

```diff
@@ -214,7 +206,7 @@ def aggregate_status(self):
                     if dt > self.timeout:
                         self.log.debug(f'{doc["host"]} reported {int(dt)} sec ago')
                         status = DAQ_STATUS.TIMEOUT
-                        if self.host_config[doc['host']] == 'tpc':
+                        if hc[doc['host']] == 'tpc':
                             if (dt > self.timeout_take_action or
                                     ((ts := self.host_ackd_command(doc['host'])) is not None and
                                      ts-now > self.timeout)):
```

Comment:
What happens here if we invoke the hypervisor because NV is timing out?

Reply:
The HV doesn't do anything because reader6 isn't in its config